index.html

```diff
@@ -21,18 +21,18 @@
         <tbody>
             <!--           first person -->
             <tr>
-                <td>Firstname Lastname</td>
-                <td>Artist Name</td>
-                <td>State Name</td>
-                <td>Number of Siblings</td>
+                <td>Emmanuel Oderemi</td>
+                <td>Johnny Drille</td>
+                <td>Ogun</td>
+                <td>3</td>
                 <td><a class="btn btn__male">Male</a></td>
             </tr>
             <!--            second person -->
```

Comment:
Hi @Shegs-Emma, you did a good work submitting the pull request, however, on bit of cleanup to do.

You only need to fill this file for yourself like you've done above. You would need to remove the details you've filled for Christianah. Do this and we should be fine to merge this.